clang/lib/CIR/CodeGen/CIRGenExprScalar.cpp

```diff
@@ -349,9 +416,8 @@ class ScalarExprEmitter : public StmtVisitor<ScalarExprEmitter, mlir::Value> {
     if (lv.isBitField()) {
       cgf.cgm.errorNYI(e->getSourceRange(), "Unary inc/dec bitfield");
       return {};
-    } else {
-      cgf.emitStoreThroughLValue(RValue::get(value), lv);
     }
+    cgf.emitStoreThroughLValue(RValue::get(value), lv);
```

Comment:
Again, I'd like to see this remain an `else` clause, because we'll want that when the case above is implemented.